### DISCUSSION TITLE
2FA SMS Country code for Colombia missing

### DISCUSSION THREAD

**Discussion by nupiter** [community/community#80978] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Feature Area

Issues

### Body

The country code for Colombia (+57) is missing in the country list for the SMS method option in 2FA.

![Screenshot from 2023-12-19 23-45-49](https://github.com/community/community/assets/20155861/14143c1b-e9d7-4e3a-8027-b4e8f21e3f95)

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @nupiter ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Projects and Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. 

If your country is not on the list, then we aren't currently able to reliably deliver text messages to your country. We track delivery rates across all countries, including ones not in the list, and review it to understand when we have to remove countries or can add them back. We do not add countries with low deliverability rates, because it leads to users being locked out of their accounts when they can’t receive the SMS for 2FA.

If GitHub doesn’t support two-factor authentication via text message for your country of residence, you must set up authentication via a TOTP application. For more information on how to configure 2FA, please review our documentation.

**Reply by fareast555** [reply]
I have the same issue with Thailand. 

Surely Thailand, which is one of the most developed countries in SE Asia, must make the list by now? And this country is now full of remote devs living in major cities like Bangkok and Chiang Mai.

**Comment by OdooClass** [comment]
It is absolutely incredible that we are now pretty much having to move off github because they can't figure out how to get this to work in Colombia.

**Comment by shinybrightstar** [comment]
Hey there @nupiter ! We know 2FA may be a new process for some folks and with that comes questions. We put together a [FAQ](https://github.com/orgs/community/discussions/68240) that will help answer your concerns above. If you still have more questions, feel free to ask them here.
